### ISSUE TITLE
torch库在前面没有 导入（3.11.4.1）

### ISSUE THREAD

**Issue by Angrybirds1** [TrickyGo/Dive-into-DL-TensorFlow2.0#33]
3.11.4.1 生成数据集
该小节，torch库在前面没有 导入，我是在本地安装了Pytorch，然后import torch，就可以成功了。

n_train, n_test, true_w, true_b = 100, 100, [1.2, -3.4, 5.6], 5
features = torch.randn((n_train + n_test, 1))
poly_features = torch.cat((features, torch.pow(features, 2), torch.pow(features, 3)), 1) 
labels = (true_w[0] * poly_features[:, 0] + true_w[1] * poly_features[:, 1]
          + true_w[2] * poly_features[:, 2] + true_b)
labels += torch.tensor(np.random.normal(0, 0.01, size=labels.size()), dtype=torch.float)

**Comment by archersama** [comment]
已修复，删去pytorch的api，换成tensorflow2的api
